diagnostics-app/src/app/app.component.css

```diff
@@ -0,0 +1,10 @@
+/**
+ * Copyright 2021 The Chromium Authors. All rights reserved.
+ * Use of this source code is governed by a BSD-style license that can be
+ * found in the LICENSE file.
+ */
```

Comment:
use // instead of /* in css files as well.

Reply:
/* */ is the only way to add comments in CSS.

https://www.xanthir.com/b4U10 According to this blog, we can have // comments in CSS but it would look like:

```
//Copyright 2021 The Chromium Authors. All rights reserved. {}
//Use of this source code is governed by a BSD-style license that can be {}
//found in the LICENSE file. {}
```

Reply:
I looked into chromium files and found out we are using /* as you did. So, let's keep it with a **leading space** preceeding the * at the beginning of each line (starting from the second line)
```
/*
 * Copyright 2021 The Chromium Authors. All rights reserved.
 * Use of this source code is governed by a BSD-style license that can be
 * found in the LICENSE file.
 */
 ```